com.unity.render-pipelines.high-definition/Documentation~/Ray-Tracing-Path-Tracing.md

```diff
@@ -61,47 +60,86 @@ You can also enable two additional settings:
 
 **Minimum Depth** set to 2, **Maximum Depth** set to 2: indirect lighting only (1 bounce)
 
-## Materials parameterization
+## How path tracing affects Material properties
+
+Path tracing changes how the following Material properties behave in your scene: 
+
+- [How transmissive objects absorb light](#surface-types)
+- [How light refracts in transmissive objects](#refraction-models)
+- [Subsurface scattering](#subsurface-scattering)
+
+This is because path tracing in HDRP implements more precise light transport simulations than rasterization. To do this, path tracing computes all lighting effects and how light interacts with Materials at the same time. This changes the appearance of Materials in path-traced scenes. For example, in the images below, the Material appears darker.
+
+The images below display the difference between transparent, double-sided materials in a rasterized and a path-traced scene:
+
+![Surface_Options](Images/HDRP_PathtracingBoxes_Raster.png)
+
+GameObjects without path tracing (rasterized).
+
+![Surface_Options](Images/HDRP_PathtracingBoxes_PathTraced.png)
+
+GameObjects with path tracing enabled.
+
+<a name="surface-types"></a>
 
-Some light phenomena can be expressed more naturally in path tracing than in rasterization, for instance:
+## Path tracing and double-sided materials
 
-- Light refraction in transmissive objects.
-- Light absorption in transmissive objects.
-- Subsurface scattering.
+When you use path tracing, the **Double-Sided** property (menu: **Inspector** > **Surface Options** > **Double-Sided**) allows transparent materials to accumulate correctly. If the you disable **Double-sided** property, rays can't exit the GameObject. 
```

Comment:
"If the you disable **Double-sided** property, rays can't exit the GameObject."
to be replaced by
"If you disable the **Double-sided** property, rays which exit the GameObject will not be properly considered as such."

Reply:
Or something like that: basically the rays will still exit, but not in the expected way, and the resulting computation will not produce the correct result.

Reply:
I replace by " rays which exit the GameObject will not behave correctly.". I will merge the PR for the cutoff, but feel free to make a new PR with a correction for this particular sentence if it isn't satisfying